src/core/dbus-mount.c

```diff
@@ -34,6 +41,27 @@ static const char *mount_get_fstype(const Mount *m) {
         return NULL;
 }
 
+static int bus_mount_method_remount(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        _cleanup_set_free_ Set *pids = NULL;
```

Comment:
`creds` and `pids` are not used.

Reply:
@yuwata Thanks, also prototype for `bus_mount_set_transient_property()` is no longer needed. I wonder why I didn't see any warning from compiler. Fixed, ptal.